src/actions/storage.js

```diff
@@ -81,6 +81,7 @@ export const uploadFile = (dispatch, firebase, { path, file, dbPath }) =>
       return firebase.database()
         .ref(dbPath)
         .push(fileData)
+        .then(fileSnap => ({ pushKey: fileSnap.key, ...fileData }))
```

Comment:
@urbantumbleweed I like the idea of having the whole snapshot available there instead of just the key under a different name.

What are your thoughts on `.then(fileSnap => ({ ...fileSnap, File: fileData }))`

This way, you can access like so:
```
uploadFile()
   .then((snapWithFile) => {
     console.log('key:', snapWithFile.key)
     console.log('file data:', snapWithFile.File) // which shouldn't run over any snapshot paramters
   })
```